packages/react-components/react-carousel-preview/library/src/components/CarouselCard/useCarouselCard.ts

```diff
@@ -19,18 +29,29 @@ export const useCarouselCard_unstable = (
   props: CarouselCardProps,
   ref: React.Ref<HTMLDivElement>,
 ): CarouselCardState => {
+  const { tabIndex } = props;
   const elementRef = React.useRef<HTMLDivElement>(null);
+  const selectPageByFocus = useCarouselContext(ctx => ctx.selectPageByFocus);
+
+  const focusAttr = useFocusableGroup({ tabBehavior: 'limited' });
+  const isFocusable = tabIndex !== undefined && tabIndex >= 0;
```

Comment:
IMO using `tabIndex` is implicit. Can we use the same approach as `Card`?

<img width="570" alt="image" src="https://github.com/user-attachments/assets/f09d43b0-f771-47f8-9c8a-d7e6d83104c0">